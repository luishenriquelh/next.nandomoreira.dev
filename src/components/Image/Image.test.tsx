import React from 'react';
import { create } from 'react-test-renderer';
import { Image } from '.';

describe('<Image />', () => {
  it('Render component', () => {
    const componentTree = create(<Image />).toJSON();
    expect(componentTree).toMatchSnapshot();
  });
});
