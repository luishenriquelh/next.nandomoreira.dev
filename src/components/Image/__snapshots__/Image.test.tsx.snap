// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> Render component 1`] = `null`;
